tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -763,6 +729,65 @@ def test_remove_environment_w_active_svcs(super_client,
     _validate_instance_removed(admin_client, service, super_client)
 
 
+def test_set_service_links(super_client, admin_client,
+                           sim_context, nsp):
+    env1 = admin_client.create_environment(name=random_str())
+    env1 = admin_client.wait_success(env1)
+
+    image_uuid = sim_context['imageUuid']
+    launch_config = {"imageUuid": image_uuid}
+
+    service1 = super_client.create_service(name=random_str(),
```

Comment:
Sorry to keep harping on this, but we had a big debate way back in early December regarding the existence of the super_client.

I understand the need to use super_client to verify resources that the regular client would not have access to. But, can't you create these services as a normal user and just use the super_clientto explicitly load/reload the restricted resource that you need to validate?